packages/suite-desktop/src-electron/modules/auto-updater.ts

```diff
@@ -183,18 +218,16 @@ const init = ({ mainWindow, store }: Dependencies) => {
         autoUpdater
             .downloadUpdate(updateCancellationToken)
             .then(() => logger.info('auto-updater', 'Update downloaded'))
-            .catch(() => logger.info('auto-updater', 'Update cancelled'));
+            .catch(() => {
+                logger.info('auto-updater', 'Update cancelled');
+            });
     });
 
+    // todo: why do we need that? this is called somewhere from react? settings? is it still relevant?
```

Comment:
There's a [corresponding method](https://github.com/trezor/trezor-suite/blob/b1e90d0712d37df9909dc380bceb19f7b1a259ac/packages/suite-desktop/src-electron/preload.ts#L64) exposed within the React app and it's used at [two](https://github.com/trezor/trezor-suite/blob/b1e90d0712d37df9909dc380bceb19f7b1a259ac/packages/suite/src/views/settings/index.tsx#L112) [places](https://github.com/trezor/trezor-suite/blob/b1e90d0712d37df9909dc380bceb19f7b1a259ac/packages/suite-desktop/src/support/DesktopUpdater/Ready.tsx#L19) to *manually* trigger a restart

Reply:
From [this](https://github.com/trezor/trezor-suite/pull/3946#discussion_r663829881) I reckon you already found out. .) Resolving.